src/fix/main.c

```diff
@@ -652,6 +652,20 @@ static ssize_t writeBytes(int fd, void *buf, size_t len)
 
 	return total;
 }
+/**
+ * @param rom0 A pointer to rom0
+ * @param startAddr What address to begin checking from
+ * @param size How many bytes to check
+ * @param areaName Name to be displayed in the warning message
+ */
+static void nonZeroWarn(uint8_t *rom0, uint16_t startAddr, uint8_t size, char const *areaName) {
```

Comment:
I would change this from `nonZeroWarn` to `warnNonZero`, to keep the verb first, like `readBytes`, `writeBytes`, `processFile`, etc.